web/impact/impact/tests/test_user_detail_view.py

```diff
@@ -628,6 +630,36 @@ def test_patch_user_with_no_profile_not_allowed_on_missing_fields(self):
             assert MISSING_PROFILE_ERROR.format(user.id) in response.data
             assert MISSING_SUBJECT_ERROR in response.data
 
+    def test_get_user_latest_active_program_location(self):
+        program = ProgramFactory(
+            program_status=ACTIVE_PROGRAM_STATUS,
+            program_family=ProgramFamilyFactory(),
+        )
+        context = UserContext(user_type=EXPERT_USER_TYPE, program=program)
+        user = context.user
+        with self.login(email=self.basic_user().email):
+            url = reverse(UserDetailView.view_name, args=[user.id])
+            response = self.client.get(url)
+            user_program = user.programrolegrant_set.order_by(
+                '-created_at'
+            ).first().program_role.program
+            program_location = response.data['latest_active_program_location']
+            assert user_program.program_family.name == program_location
```

Comment:
This smells funny. Shouldn't we be testing program family location against the received location instead?

Reply:
`program_location = response.data['latest_active_program_location']` is the received location which i'm testing against